server/routes/api/contest.js

```diff
@@ -0,0 +1,104 @@
+const express = require("express");
+const contestRouter = express.Router();
+
+const auth = require("../../middleware/auth");
+const ContestModel = require("../../models/contest");
+
+// Route: GET api/contests/
+// Desc: get all active contests
+// access:  private
+
+contestRouter.get("/contests", auth, (req, res) => {
```

Comment:
I think this should be its own api route -> `/api/contests` instead of `/api/contest/contests`

Reply:
Updated, I was afraid to include separate router for contests, but as you said the URLs are cleaner this way.